### PR TITLE
119 like component styling

### PR DESCRIPTION
**Pull request by therearedoors**
- improved layout
- added button colour detail

Like before/after:
image

image

### CODE REVIEW

**Review by Auenc — CHANGES_REQUESTED** [review]
Looks like there is some double quotes in some files. 

Would you be able to make sure that you're only using single quotes, and make sure each line of JS has a semi-colon on the end please?

File: src/components/posts/PostForm.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const PostForm = ({ handleSubmit, handleChange, inputValue }) => {
           name='content'
           onChange={handleChange}
         />
-        <Button type='submit' variant='contained'>
+        <Button type="submit" variant="contained">
```
Comment (Auenc) [inline src/components/posts/PostForm.js:20]: Double quotes here

**Review by Auenc — DISMISSED** [review]
If you can change the component that has double quotes, this will be good to go!

File: src/components/posts/style.css [hunk 2]
```diff
@@ -1,34 +1,39 @@
...
-  padding-top: 2.5rem;
+    display: flex;
+    justify-content: space-evenly;
+    padding-top: 2.5rem;
 }
 
 .posts-section {
-  flex: 1;
-  margin-top: 15px;
+    margin-top: 15px;
```
Comment (mikemherron) [inline src/components/posts/style.css:8]: Watch formatting and spacing in here, looks different from the rest of the file. I belive the convention is 2 spaces for tabs.

File: src/components/posts/PostLike.js [hunk 3]
```diff
@@ -7,61 +7,61 @@ import colours from '../../utils/colours';
...
 
-  function likePost(event){
-  updateTalkStatus(true);
-  client.post(`/post/${postId}/like/`)
-  .then(res => {
-    updateId(_ => res.data.data.id);
-    updateLikes(likes => [res.data.data, ...likes]);
-    updateStyle({backgroundColor: blue});
+function likePost(event){
+setTalkStatus(true);
```
Comment (mikemherron) [inline src/components/posts/PostLike.js:27]: Where has the indentation gone!

**Review by mikemherron — APPROVED** [review]
Couple of formatting things to fix. Same point as last PR - in fact, I think the ";" added here are the same as the last PR, try not to do too many unrelated changes in a single PR as makes it hard to see what the actual signficant change was (both in PR and when looking back at commit history).

**Review by elvisonob — APPROVED** [review]

**Pull request merged**